### DISCUSSION TITLE
Nightly builds have been failing recently

### DISCUSSION THREAD

**Discussion by An-anonymous-coder** [FreeTubeApp/FreeTube#6100] [Q&A] [answered]
The [nightly builds](https://github.com/FreeTubeApp/FreeTube/actions/workflows/build.yml) have been failing for the past 4 days. Is something wrong?

**Comment by ChunkyProgrammer** [comment, marked as answer]
Yes, but it should get fixed when this PR gets merged: https://github.com/FreeTubeApp/FreeTube/pull/6084
